js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/RunTests.scala

```diff
@@ -95,50 +95,19 @@ private[test] class RunTests(config: JSEnvSuiteConfig, withCom: Boolean) {
     }
   }
 
-  @Test // Node.js strips double percentage signs - #500
+  // #500 Node.js used to strip double percentage signs even with only 1 argument
+  @Test
   def percentageTest: Unit = {
-    val counts = 1 to 15
-    val argcs  = 1 to 3
-    val strings = counts.map("%" * _)
-
-    val strlists = for {
-      count  <- argcs
-      string <- strings
-    } yield List.fill(count)(string)
-
-    val codes = for {
-      strlist <- strlists
-    } yield {
-      val args = strlist.map(s => s""""$s"""").mkString(", ")
-      s"console.log($args);\n"
-    }
+    val strings = (1 to 15).map("%" * _)
+    val code = strings.map(str => s"""console.log("$str");\n""").mkString("")
+    val result = strings.mkString("", "\n", "\n")
 
-    val result = strlists.map(_.mkString(" ") + "\n").mkString("")
-
-    withRun(codes.mkString("")) {
+    withRun(code) {
       _.expectOut(result)
         .closeRun()
     }
   }
```

Comment:
Could you refactor this in a separate commit so we don't accidentally introduce a regression?